src/app/shared/dovetail/dovetail-audio.ts

```diff
@@ -37,13 +37,14 @@ const CAN_PLAY = 'canplay';
 const CAN_PLAY_THROUGH = 'canplaythrough';
 
 const PROXIED_EVENTS = [
-  TIME_UPDATE, SEEKED, ERROR, PLAYING, ABORT, CAN_PLAY, CAN_PLAY_THROUGH
```

Comment:
Error events are now manually handled by this class, and only propagate up if we can't handle them.  (Catching the 406 here).